applicationinsights.dev.json

```diff
@@ -13,20 +13,21 @@
   "selfDiagnostics": {
     "destination": "console"
   },
-  "preview": {
-    "sampling": {
-      "overrides": [
-        {
-          "attributes": [
-            {
-              "key": "http.url",
-              "value": "https?://[^/]+/health.*",
-              "matchType": "regexp"
-            }
-          ],
-          "percentage": 100
-        }
-      ]
-    }
+  "sampling": {
+    "requestsPerSecond": 5.0,
+    "overrides": [
+      {
+        "telemetryType": "request",
+        "attributes": [
+          {
+            "key": "http.url",
+            "value": "https?://[^/]+/health.*",
+            "matchType": "regexp"
+          }
+        ],
+        "percentage": 0
```

Comment:
Should this be set to 0 rather than 100? Also this file appears to be used for all environments and the other one for nothing so maybe we can just delete this and use `applicationinsights.json` everywhere?

Reply:
That environment variable is also set to `applicationinsights.dev.json` in `values-preprod.yaml` and `values-prod.yaml` so it looks like we are currently recording 100% of health checks in all environments which seems like a bit much...